joblib/test/test_numpy_pickle.py

```diff
@@ -230,27 +200,27 @@ def test_numpy_persistence():
 
 
 @with_numpy
-def test_numpy_persistence_bufferred_array_compression():
+def test_numpy_persistence_bufferred_array_compression(tmpdir):
     big_array = np.ones((_IO_BUFFER_SIZE + 100), dtype=np.uint8)
-    filename = env['filename'] + str(random.randint(0, 1000))
+    filename = tmpdir.join('test.pkl').strpath + str(random.randint(0, 1000))
```

Comment:
Just checked it is a function fixture. So I think most of the `str(random.randint(0, 1000))` can go.

Inside a single test function, you may want to to use different filenames:
```py
filename1 = tmpdir.join('test1.pkl').strpath
...
filename2 = tmpdir.join('test2.pkl').strpath
...
```
although when the variable `filename` is reused you can probably only define it once and reuse it.
```